docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,382 @@
+      resources:
+        limits:
+          cpu: 7
+          memory: 56G
+      placement:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/zone
+                operator: In
+                values:
+                - us-east-1b
+              - key: scylla.scylladb.com/node-type
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+    - name: c
+      members: 1
+      storage:
+        storageClassName: scylladb-local-xfs
+        capacity: 1800G
+      resources:
+        limits:
+          cpu: 7
+          memory: 56G
+      placement:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/zone
+                operator: In
+                values:
+                - us-east-1c
+              - key: scylla.scylladb.com/node-type
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+```
+
+Apply the manifest:
+```console
+kubectl --context "${CONTEXT_DC1}" apply --server-side -f dc1.yaml
+```
+
+Wait for to the cluster to be fully rolled out and all of its nodes to be in state UN. You can verify that by running: 
+```console
+kubectl --context "${CONTEXT_DC1}" -n scylla exec -it pod/scylla-cluster-us-east-1-a-0 -c scylla -- nodetool status
+```
+
+The expected output should look similar to the below:
+```console
+Datacenter: us-east-1
+=====================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load       Tokens       Owns    Host ID                               Rack
+UN  10.0.70.195  290 KB     256          ?       494277b9-121c-4af9-bd63-3d0a7b9305f7  c
+UN  10.0.59.24   559 KB     256          ?       a3a98e08-0dfd-4a25-a96a-c5ab2f47eb37  b
+UN  10.0.19.237  107 KB     256          ?       64b6292a-327f-4128-852a-6004039f402e  a
+```
+
+Use the below commands and their expected outputs as a reference for retrieving the PodIPs used by the cluster for inter-node communication.
+```console
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-a-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.19.237
+
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-b-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.59.24
+
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-c-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.70.195
+```
+
+We are going to utilize the retrieved addresses as seeds for the other datacenter.
```

Comment:
We should make a note here that PodIPs are ephemeral, so users should consider using DNS that would make sure they are up to date. You can mention you skipped it for the simplicity of this document.